### ISSUE TITLE
White Screen

### ISSUE THREAD

**Issue by kaiw1001** [Maschell/StreamingPluginClient#4]
Whenever I connect to my Wii U, I get a white screen, when I use it using console, it says timed out

**Comment by MisterSilkySmooth** [comment]
Yeah, I'm having the same problem. All I get is a white screen when attempting to stream to my PC.
